covasim/webapp/index.html

```diff
@@ -271,16 +273,16 @@
 
     <!-- Show the actual graphs -->
     <div id="graphs" class="results" v-bind:class="{ empty: result.graphs.length === 0 }">
-        <template v-if="result.graphs.length === 0">
-          <p class="h3"><i>Click "Run" to display graphs</i></p>
-        </template>
-        <template v-else>
-          <div class="row no-gutter">
-            <div class="col-md-6" v-for="x in result.graphs">
-              <plotly-chart :graph="x" :key="x.id"/>
-            </div>
+      <template v-if="result.graphs.length === 0">
+        <p class="h5"><i>Click "Run" to display graphs</i></p>
```

Comment:
Text was too large on my screen

Reply:
You should really make separate issues for these and document your screen size. Makes it easier for us to validate your changes.

Reply:
We can also ask the user group for feedback on this